ios/Example/Tests/SecureEnclaveDigitalSignatureKeyManagerTests.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  SecureEnclaveDigitalSignatureKeyManagerTests.swift
+//  SecuritySdk_Tests
+//
+
+import XCTest
+import SecuritySdk
+
+final class SecureEnclaveDigitalSignatureKeyManagerTests: XCTestCase {
+    lazy var manager = SecureEnclaveDigitalSignatureKeyManager(
+        tag: "app.cash.trifle.s2dk.keys.digital_signature"
+    )
```

Comment:
nit for future: trifle has effectively replaced s2dk, so we should. just get rid of s2dk eveywhere